tests/app/main/test_suppliers.py

```diff
@@ -662,6 +662,26 @@ def test_recovery_supplier_sees_g12_links(self, framework_interest, on_framework
                 u3="/suppliers/frameworks/g-cloud-12",
             )
 
+    def test_recovery_supplier_sees_banner(self):
+        self.data_api_client.get_supplier.return_value = get_supplier(id='577184')  # Test.DM_G12_RECOVERY_SUPPLIER_IDS
```

Comment:
I think thee id is an integer, instead of string. Either way, better to test with both types.

Reply:
See https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/1288/ - we handle it being either a string or integer.

Reply:
I've switched to integers.